packages/doc/content/components/components/checkbox/switch-web.mdx

```diff
@@ -5,8 +5,13 @@ Switches are best used for communicating activation (e.g., on/off states), while
 
 ### Usage
 
-```javascript reactLive=true center=true
-<Checkbox.Switch />
+```javascript reactLive=true center=true state=true
+render(() => {
+  const [checked, setChecked] = useState(false);
```

Comment:
```suggestion
  const [checked, toggleChecked] = useState(false);
```

What do you think of changing `set` to `toggle` on functions that change boolean values?

Reply:
Done!